backends/p4tools/testgen/backend/tf.h

```diff
@@ -0,0 +1,44 @@
+#ifndef TESTGEN_BACKEND_TF_H_
+#define TESTGEN_BACKEND_TF_H_
+
+#include <cstddef>
+
+#include <boost/optional/optional.hpp>
+
+#include "lib/cstring.h"
+
+#include "backends/p4tools/testgen/lib/test_spec.h"
+
+namespace P4Tools {
+
+namespace P4Testgen {
+
+/// Extracts information from the @testSpec to emit a TF test case.
```

Comment:
TF = Test Framework?

Reply:
This comment seems wrong to me. Isn't this a base class that should be overridden? If that's correct the constructor should be protected.